examples/workflows/plot_simulate_beta.py

```diff
@@ -0,0 +1,234 @@
+underlying beta mediated sensory suppression.
+"""
+
+# Authors: Nick Tolley <nicholas_tolley@brown.edu>
+
+from hnn_core import simulate_dipole, law_2021_model, jones_2009_model
+from hnn_core.viz import plot_dipole
+
+###############################################################################
+# We begin by instantiating the network model from Law et al. 2021 [1]_.
+net = law_2021_model()
+
+###############################################################################
+# The Law 2021 model is based on the network model described in
+# Jones et al. 2009 [2]_ with several important modifications. One of the most
+# significant changes is substantially increasing the rise and fall time
+# constants of GABAb-conductances on L2 and L5 pyramidal. Another important
+# change is the removal of calcium channels from basal dendrites and soma of
+# L5 pyramidal cells specifically.
+# We can inspect these properties with the ``net.cell_types`` attribute which
+# contains information on the biophysics and geometry of each cell.
+net_jones = jones_2009_model()
+
+jones_rise = net_jones.cell_types['L5_pyramidal'].p_syn['gabab']['tau1']
+law_rise = net.cell_types['L5_pyramidal'].p_syn['gabab']['tau1']
+print(f'GABAb Rise (ms): {jones_rise} -> {law_rise}')
+
+jones_fall = net_jones.cell_types['L5_pyramidal'].p_syn['gabab']['tau2']
+law_fall = net.cell_types['L5_pyramidal'].p_syn['gabab']['tau2']
+print(f'GABAb Fall (ms): {jones_fall} -> {law_fall}\n')
+
+print('Apical Dendrite Channels:')
+print(net.cell_types['L5_pyramidal'].p_secs['apical_1']['mechs'].keys())
+print("\nBasal Dendrite Channels ('ca' missing):")
+print(net.cell_types['L5_pyramidal'].p_secs['basal_1']['mechs'].keys())
+
+###############################################################################
+# A major change to the Jones 2009 model is the addition of a
+# Martinotti-like recurrent tuft connection [3]_. This new connection
+# originates from L5 basket cells, and provides GABAa inhibition on
+# the distal dendrites of L5 basket cells.
+print('Recurrent Tuft Connection')
+print(net.connectivity[16])
+
+###############################################################################
+# The remaining changes to the connectivity was the removal of an
+# L2_basket -> L5_pyramidal GABAa connection, and replacing it with GABAb.
+print('New GABAb connection')
+print(net.connectivity[15])
+
+print('\nConnection Removed from Law Model')
+print(net_jones.connectivity[10])
+
+###############################################################################
+# To demonstrate sensory depression, we will add the drives necessary to
+# generate and ERP similar to
+# :ref:`evoked example <sphx_glr_auto_examples_plot_simulate_evoked.py>`,
+# but modified to reflect the parameters used in Law et al. 2021.
+# Specifically, we are considering the case where a tactile stimulus is
+# delivered at 150 ms. 25 ms later, the first input to sensory cortex arrives
```

Comment:
I think it would be nice to change the time axis so that 150 ms = 0 when plotting the dipole. ERP researchers normally name these waveforms ... N20, P50 etc. based on the latencies. I was trying to figure out which one you're trying to simulate but I'm not sure ... perhaps @cjayb has more opinions

Reply:
I can do this in a hacky matplotlib hacky way for this example, but maybe this is an API issue? Like in #315 setting a `tstart` that is negative should allow for this sort of plotting.

Reply:
okay yes, let's avoid hacky solutions. Could you raise an issue and I'll see if it can be addressed as part of #315 ?